src/components/Carousel.js

```diff
@@ -623,6 +624,7 @@ export default class Carousel extends Component {
           className="BrainhubCarousel__arrows BrainhubCarousel__arrowLeft"
           onClick={this.prevSlide}
           ref={el => this.arrowLeftNode = el}
+          disabled={value <= 0}
```

Comment:
`disabled` prop should be set to true only if `infinite` option is not enabled. Please add checking for it here.